stdpopsim/catalog/CaeEle/genetic_maps.py

```diff
@@ -0,0 +1,32 @@
+import stdpopsim
+
+_species = stdpopsim.get_species("CaeEle")
+
+_gm = stdpopsim.GeneticMap(
+    species=_species,
+    id="RockmanRIAIL",
```

Comment:
One of the failing tests is complaining about the `id` used for the genetic map. See the "naming conventions" section of the dev docs: https://stdpopsim.readthedocs.io/en/latest/development.html#naming-conventions
The genetic map should be using coordinates that match a specific genome assembly, so the naming convention asks for the genome assembly to be part of the genetic map id.